groups/groups.yaml

```diff
@@ -263,6 +264,7 @@ groups:
     settings:
       ReconcileMembers: "true"
     members:
+      - k8s-infra-release-admins@kubernetes.io
```

Comment:
Mostly fine with this (SIG Release ownership + appropriately paranoid admins), though I would like to refresh my memory on what this allows

Reply:
Artifact admins seem to get...

GCS Admin access:
https://github.com/kubernetes/k8s.io/blob/ed671b680ce43ecc024d56df224bb85a8ce1a36e/infra/gcp/lib.sh#L26-L28

https://github.com/kubernetes/k8s.io/blob/ed671b680ce43ecc024d56df224bb85a8ce1a36e/infra/gcp/lib.sh#L294-L308

https://github.com/kubernetes/k8s.io/blob/ed671b680ce43ecc024d56df224bb85a8ce1a36e/infra/gcp/lib.sh#L310-L323

https://github.com/kubernetes/k8s.io/blob/ed671b680ce43ecc024d56df224bb85a8ce1a36e/infra/gcp/ensure-prod-storage.sh#L117-L121

https://github.com/kubernetes/k8s.io/blob/ed671b680ce43ecc024d56df224bb85a8ce1a36e/infra/gcp/ensure-prod-storage.sh#L155-L156

https://github.com/kubernetes/k8s.io/blob/ed671b680ce43ecc024d56df224bb85a8ce1a36e/infra/gcp/ensure-prod-storage.sh#L201-L202

https://github.com/kubernetes/k8s.io/blob/ed671b680ce43ecc024d56df224bb85a8ce1a36e/infra/gcp/ensure-prod-storage.sh#L207-L208

https://github.com/kubernetes/k8s.io/blob/ed671b680ce43ecc024d56df224bb85a8ce1a36e/infra/gcp/ensure-prod-storage.sh#L211-L220

https://github.com/kubernetes/k8s.io/blob/ed671b680ce43ecc024d56df224bb85a8ce1a36e/infra/gcp/ensure-prod-storage.sh#L390-L392

---

Cloud Run access to deploy the auditor:
https://github.com/kubernetes/k8s.io/blob/ed671b680ce43ecc024d56df224bb85a8ce1a36e/infra/gcp/ensure-prod-storage.sh#L315-L324

Reply:
Thanks, crawling through this was fun.  

Granting iam.serviceAccountUsers for the auditor service account means this group can also pick up whatever that service account can do
https://github.com/kubernetes/k8s.io/blob/07a3a07626d8733550866abe799cf58241045a18/infra/gcp/lib.sh#L348-L353

Looks like mostly related to logging though https://github.com/kubernetes/k8s.io/blob/07a3a07626d8733550866abe799cf58241045a18/infra/gcp/lib.sh#L402-L412